include/dlaf/eigensolver/tridiag_solver/kernels.h

```diff
@@ -0,0 +1,292 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2022, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+#include "dlaf/common/callable_object.h"
+#include "dlaf/eigensolver/tridiag_solver/coltype.h"
+#include "dlaf/matrix/tile.h"
+#include "dlaf/types.h"
+
+#ifdef DLAF_WITH_GPU
+#include <cusolverDn.h>
```

Comment:
I think you want to do this instead here:
```suggestion
#include <dlaf/gpu/lapack/api.h>
```